### DISCUSSION TITLE
Double-contributor listing for Renovate

### DISCUSSION THREAD

**Discussion by TWiStErRob** [renovatebot/renovate#20112] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

When I enable Renovate I immediately get two extra contributors on all my repos. Why are the names not matching up?

![image](https://user-images.githubusercontent.com/2906988/215505302-9e59b659-e93b-487e-aa8f-5109cf3df16e.png)

If their names were matching only one entry would show (guess). Reducing noise and confusion.

It looks like commits are authored by: `renovate[bot]` https://github.com/marketplace/renovate
and PRs are opened by: `renovate` https://github.com/apps/renovate

**Comment by rarkins** [comment]
New repos should see only one identity, old repos can't do anything about it (or any other existing committer).

Historically GitHub apps/bots had no way to sign commits unless you signed them with a "real" user account. So `renovate[bot]` was the app's identify while `renovate-bot` is the real user which signed commits. Nowadays you should see the bot signing its own commits, as GitHub made that an available feature through the API and we now use it.

Example of old commit: https://github.com/renovatebot/renovate/pull/2481/commits
Example of new commit: https://github.com/renovatebot/renovate/pull/20103/commits

**Reply by TWiStErRob** [reply]
Makes sense, I thought I saw "renovate authored and renovate committed" in the past, but couldn't find those now. Thanks for the clarification!
